tests/render/test_render_BulletListContentBlock.py

```diff
@@ -69,75 +64,18 @@ def test_parse_row_condition_string_pandas_engine():
-    assert len(test_files) == 61, (
-        "Something went wrong when collecting JSON Expectation test fixtures"
-    )
-
-    # Not ported over to V1 so ignore for purposes of this test
-    UNSUPPORTED_EXPECTATIONS = {
-        "expect_column_pair_cramers_phi_value_to_be_less_than",
-        "expect_column_parameterized_distribution_ks_test_p_value_to_be_greater_than",
-        "expect_column_chisquare_test_p_value_to_be_greater_than",
-    }
-
-    # Loop over all test_files, datasets, and tests:
-    test_results = defaultdict(list)
-    for filename in test_files:
-        test_definitions = json.load(open(filename))
-        if test_definitions["expectation_type"] in UNSUPPORTED_EXPECTATIONS:
-            continue
-        for dataset in test_definitions["datasets"]:
-            for test in dataset["tests"]:
-                # Construct an expectation from the test.
-                if type(test["in"]) == dict:  # noqa: E721 # FIXME CoP
-                    # Skip tests with invalid configurations
-                    if test["in"].get("catch_exceptions"):
-                        continue
-                    fake_expectation = ExpectationConfiguration(
-                        type=test_definitions["expectation_type"],
-                        kwargs=test["in"],
-                    )
-                else:
-                    # This would be a good place to put a kwarg-to-arg converter
-                    continue
-
-                # Attempt to render it
-                render_result = ExpectationSuiteBulletListContentBlockRenderer.render(
-                    [fake_expectation]
-                )
-                assert render_result is not None
-                render_result = render_result.to_json_dict()
-
-                assert isinstance(render_result, dict)
-                assert "content_block_type" in render_result
-                assert render_result["content_block_type"] in render_result
-                assert isinstance(render_result[render_result["content_block_type"]], list)
-
-                # TODO: Assert that the template is renderable, with all the right arguments, etc.
-                # rendered_template = pTemplate(el["template"]).substitute(el["params"])
-
-                test_results[test_definitions["expectation_type"]].append(
-                    {
-                        test["title"]: render_result,
-                        # "rendered_template":rendered_template
-                    }
-                )
-
-    # TODO: accommodate case where multiple datasets exist within one expectation test definition
-    with open(
-        file_relative_path(
-            __file__,
-            "./output/test_render_bullet_list_content_block.json",
-            strict=False,
-        ),
-        "w",
-    ) as f:
-        json.dump(test_results, f, indent=2)
+def test_expectations_using_expectation_definitions():
```

Comment:
I believe the purpose of this test was to a specific render for all expectations that were defined using `test_expectations_v3_api.py`. Since that testing strategy was hard to maintained and didn't contain all the expectations, this strategy isn't ideal. For the purpose of this PR, I extracted all the expectations that were under test and moved them into `BulletListContentBlock.json`. This let me remove all the filtering and I can not iterate through them and run the asserts in this test. This should be equivalent to what was being run before.

Reply:
It's already out of date since we've already moved away from this json testing framework. I think we should eventually delete this method and incorporate this test in our new expectation testing framework. I will file a followup ticket.
I think it's outside the scope of this work since it's currently falling out of date and this doesn't make it worse. I think it's a good callout to fix it.

Reply:
The ticket is: https://greatexpectations.atlassian.net/browse/GX-850